main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
# is used to comment out lines in Python - this also applies to lines 5, 12, and 22